BUILD.bazel

```diff
@@ -81,3 +84,52 @@ pkg_tar(
     strip_prefix = "./rules_nodejs_package",
     tags = ["manual"],
 )
+
+BAZEL_VERSION = "0.28.1"
+
+BAZEL_BINARY = "@build_bazel_bazel_%s//:bazel_binary" % BAZEL_VERSION.replace(".", "_")
+
+NPM_PACKAGES = {
+    "//packages/jasmine:npm_package": "@bazel/jasmine",
+    "//packages/karma:npm_package": "@bazel/karma",
+    "//packages/labs:npm_package": "@bazel/labs",
+    "//packages/protractor:npm_package": "@bazel/protractor",
+    "//packages/stylus:npm_package": "@bazel/stylus",
+    "//packages/typescript:npm_package": "@bazel/typescript",
+}
+
+bazel_integration_test(
+    name = "e2e_jasmine",
+    bazel_binary = BAZEL_BINARY,
+    # some bazelrc imports are outside of the nested workspace so
+    # the test runner will handle these as special cases
+    bazelrc_imports = {
+        "//:common.bazelrc": "import %workspace%/../../common.bazelrc",
+    },
+    # package.json will be updated with `file:` links to the absolute paths
+    # of the generated npm packages in runfiles
+    npm_packages = NPM_PACKAGES,
+    # replace the following repositories with the generated archives
+    repositories = {
+        "//:release": "build_bazel_rules_nodejs",
+    },
+    workspace_files = "@e2e_jasmine//:all_files",
+    workspace_under_test = "e2e_jasmine",
+)
+
+bazel_integration_test(
+    name = "e2e_angular_bazel_example",
+    size = "large",
+    timeout = "long",
```

Comment:
I'm excited to have `flaky_test_attempts=2` around all these tests that rely on package installs